### DISCUSSION TITLE
question: how can I reproduce behavior from vim?: `ci"`

### DISCUSSION THREAD

**Discussion by bkielbasa** [helix-editor/helix#7850] [Q&A]
Hello,

I very often use the command `ci"` when I want to change a text that's inside in `""` in the current line. In Vim, there's a simple way to do it. In Helix it seems a bit more complicated. To achieve something similar I have to type `f"lmi"c` what's much more complicated and error probe.

**Comment by ymgyt** [comment]
```
"foo"
 ↑ cursor
```
type `mi"c`

**Comment by bkielbasa** [comment]
@ymgyt thanks for your response. The problem is that I have to navigate to it, that's why for now I type `f"lmi"c` (the first part is navigation) and the question is if I can do it without the navigation, like in Vim

**Comment by Brixy** [comment]
Your questions seems to be related to [#5903](https://github.com/helix-editor/helix/issues/5903)

For

```
a (b)
```

```
f(mi(c
```

works like in Vim.

**Reply by robgunnatkin** [reply]
It doesn't work with "

```
func wtf("string")
    ↑ cursor
```

In vim `ci"`

```
func wtf("")

```

Using the same top cursor position above

`f"mi"` selects nothing
`f"miw` selects only **("**

Yet, `f(mi(` works like you would expect.

Behavior seems inconsistent to me especially around core concepts of `fFtT`
